actions/new.action.ts

```diff
@@ -674,37 +685,39 @@ const copyEnvFile = async (dir: string, envExample: string, envFile: string) =>
 };
 
 const createRegistry = async (
-  dir: string,
-  shouldInitializePrisma: boolean,
-  shouldInitializeUserService: boolean,
-  shouldInstallMonitoring: boolean,
-  shouldInitializeMonitoring: boolean,
-  shouldInitializeTemporal: boolean,
-  shouldInitializeLogging: boolean,
-  shouldInitializeFileUpload: boolean
+  projectDirectory: string,
+  spec: {
+    projectName: string,
+    prismaSetup: boolean,
+    userServiceSetup: boolean,
+    monitoringSetup: boolean,
+    monitoringServicesSetup: boolean,
+    temporalSetup: boolean,
+    loggingSetup: boolean,
+    fileUploadSetup: boolean,
+    packageManager: string
+  }
 ): Promise<void> => {
-  const filePath = join(process.cwd(), dir, '.stencil');
-  
-  const setupInfo = [
-    shouldInitializePrisma ? 'Prisma Setup' : '',
-    shouldInitializeUserService ? 'User Services Setup' : '',
-    shouldInstallMonitoring ? 'Monitoring Installed' : '',
-    shouldInitializeMonitoring ? 'Monitoring Setup' : '',
-    shouldInitializeTemporal ? 'Temporal Setup' : '',
-    shouldInitializeLogging ? 'Logging Setup' : '',
-    shouldInitializeFileUpload ? 'File Upload Setup' : ''
-  ].filter(info => info !== '').join('\n');
+  const { projectName, packageManager, ...setupInfo } = spec;
 
+  const filePath = join(process.cwd(), projectDirectory, 'spec.yaml');
+  
   try {
     await fs.promises.access(filePath, fs.constants.F_OK);
-    console.log('.stencil file already exists');
+    console.log('spec.yaml file already exists');
```

Comment:
either convert this to a proper logger or remove.

Reply:
What do you mean by proper logger?
Like using chalk and passing Message parameter?

Reply:
yes